skbio/stats/power.py

```diff
@@ -942,22 +1014,17 @@ def _calculate_power_curve(test, samples, sample_counts, ratio=None,
     num_iter : int
         The default is 1000. The number of p-values to generate for each point
         on the curve.
-
```

Comment:
@jairideout doc whisperer, is it OK to remove empty lines between sections in the docstrings?